SU2_CFD/include/solvers/CEulerSolver.hpp

```diff
@@ -1026,12 +1028,39 @@ class CEulerSolver : public CFVMFlowSolverBase<CEulerVariable, ENUM_REGIME::COMP
    */
   void InitTurboContainers(CGeometry *geometry, CConfig *config) final;
 
+  /*!
+   * \brief Initilize turbomachinery performance shared_ptr.
+   * \param[in] geometry - Geometrical definition of the problem.
+   * \param[in] config - Definition of the particular problem.
+   */
+  void InitTurboPerformance(CGeometry *geometry, CConfig** config) final;
+  
+  /*!
+   * \brief Get Primal variables for turbo performance computation
+   *        iteration can be executed by multiple threads.
+   * \return returns Density, pressure and TurboVelocity (IN/OUTLET)
+   */
+  inline vector<su2double> GetTurboPrimitive(unsigned short iBlade, unsigned short iSpan, bool INLET) override {
+    TurboPrimitive.clear();
+    if (INLET) {
+      TurboPrimitive.push_back(DensityIn[iBlade][iSpan]); TurboPrimitive.push_back(PressureIn[iBlade][iSpan]);
+      TurboPrimitive.push_back(TurboVelocityIn[iBlade][iSpan][0]);TurboPrimitive.push_back(TurboVelocityIn[iBlade][iSpan][1]);
```

Comment:
Please fix the formatting
```suggestion
      TurboPrimitive.push_back(DensityIn[iBlade][iSpan]);
      TurboPrimitive.push_back(PressureIn[iBlade][iSpan]);
      TurboPrimitive.push_back(TurboVelocityIn[iBlade][iSpan][0]);
      TurboPrimitive.push_back(TurboVelocityIn[iBlade][iSpan][1]);
```